tools/check.py

```diff
@@ -80,7 +89,7 @@ def _validate_no_fixme(self):
         valid = True
         for i, line in enumerate(self.markdown.splitlines()):
             if re.search("FIXME", line, re.IGNORECASE):
-                logging.error(
+                report_error("error",
```

Comment:
`report_error("error"` seems redundant to me; perhaps the function should just be called `report`? Or shouldn't take in the type. I think it does make sense for some style things down the road to raise a warning rather than an error though; `report("warning"` seems sensible to me.